src/engine/ruleset/decoders/f5/f5-apm-own.yml

```diff
@@ -0,0 +1,451 @@
+---
+name: decoder/f5-apm-own/0
+
+metadata:
+  module: F5
+  title: F5 APM Decoder for own providers
+  description: Decodes HEADER F5 BIG-IP Access Policy Manager logs of own providers.
+  compatibility: >
+    TBD
+  author:
+    name: Wazuh, Inc.
+    date: 2023/01/10
+  references:
+    - TBD
+
+sources:
+  - decoder/queue-syslog/0
+  - decoder/queue-localfile/0
+
+parse:
+  logpar:
+    - event.original: <~> <@timestamp/%Y\/%m\/%d %T> <~> <log.level> <~payload>
+
+parse:
+  logpar:
+    - ~payload: <~>[<process.pid>]<~/ignore/:> <event.provider>:<~/ignore/ ><~content>
+    - ~payload: <~>[<process.pid>]<~/ignore/:> <event.provider>:<~/ignore/:><~content>
+    - ~payload: <~>[<process.pid>]<~/ignore/:> <event.provider>:<~>:<~/ignore/ ><~content>
+    - ~payload: <~>:<~/ignore/:> <event.provider>:<~/ignore/:><~content>
+
+normalize:
+  # 011f0005
+  # March 2017/03/04 11:21:59 unte very-high ueipsa[748]: 011f0005: :cti: failure (Client side: vip=https://www5.example.com/olli/rever.html?rsp=oluptat#metco profile=ipv6-icmp pool=edolorin client_ip=10.104.110.134)
+  - check: event.provider==011f0005
+    logpar:
+      - ~content: <~/ignore/:><~>:<~/ignore/ ><~tmp.rsa_misc_result> <~/literal/(Client side:><~/ignore/ >vip=<~url/uri> profile=<network.protocol> pool=<~> client_ip=<source.ip>\)
+    map:
+      - rsa.internal.messageid : $event.provider
+      - event.code : $event.provider
+      - rsa.misc.log_session_id : $~tmp.log_session_id
+      - url : $~url
+      - ~url : +ef_delete
+
+
+  # 01490106
+  # oremagn[2121]: 01490106: :itseddo: uptatev: AD module: authentication with 'oditem' failed in allow: Preauthentication failed, principal name: inimaven. failure olor
+  - check: event.provider==01490106
+    logpar:
+      - ~content: <~tmp.log_session_id>:<~/ignore/:> <~>:<~/ignore/:> <~>'<~tmp.user>' <~>:<~/ignore/:> <~>, <~>. <~tmp.rsa_misc_result> <~>
+    map:
+      - rsa.internal.messageid : $event.provider
+      - event.code : $event.provider
+      - related.user : $~tmp.user
+      - user.name : $related.user
+      - rsa.misc.result : $~tmp.rsa_misc_result
+      - rsa.misc.log_session_id : $~tmp.log_session_id
```

Comment:
These ones also are repeated many times, can we map them all at the beginning of the normalization?